### PR TITLE
Added wintery skins to content

### PR DESCRIPTION
**Pull request by TheHollidayInn**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes put_issue_url_here

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

This PR adds the winter skins to our content list

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID:

### CODE REVIEW

File: features.config.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
+{
```
Comment (paglias) [inline features.config.json]: was this meant to be in the PR?

Comment (SabreCat) [reply features.config.json]: That looks like the enterprise plans config file. Wayward `git add -A`?

Comment (TheHollidayInn) [reply features.config.json]: UGhh, my bad

File: website/common/script/content/appearance/skin.js [hunk 2]
```diff
@@ -64,5 +64,14 @@ module.exports = prefill({
   'shark': {price: 2, set: sets.splashySkins},
   'tropicalwater': {price: 2, set: sets.splashySkins},
 
+  'aurora': {price: 2, set: sets.winterySkins},
+  'dapper': {price: 2, set: sets.winterySkins},
+  'festive': {price: 2, set: sets.winterySkins},
+  'holly': {price: 2, set: sets.winterySkins},
+  'polar': {price: 2, set: sets.winterySkins},
+  'sugar': {price: 2, set: sets.winterySkins},
+  'snowy': {price: 2, set: sets.winterySkinColors},
```
Comment (khipkin) [inline website/common/script/content/appearance/skin.js]: Are snowy and winterstar intentionally different from the rest?

**Review by SabreCat — CHANGES_REQUESTED** [review]
Should be `winterySkins` instead of `winterySkinColors` on those last couple of lines. Good to go after that!

**Comment by TheHollidayInn** [comment]
Fixed. Sorry for the delay.

**Review by SabreCat — APPROVED** [review]

**Pull request merged**